frontend/src/app.js

```diff
@@ -0,0 +1,43 @@
+
+import 'babel-polyfill/browser';
+import es6Promise from 'es6-promise';
+import 'isomorphic-fetch';
+import 'l20n/dist/compat/web/l20n';
+import Raven from 'raven-js';
+
+import './app/lib/ga-snippet';
+import config from './app/config';
+
+import error from './pages/error.js';
+import experiment from './pages/experiment.js';
+import experiments from './pages/experiments.js';
+import home from './pages/home.js';
+import legacy from './pages/legacy.js';
+import onboarding from './pages/onboarding.js';
+import retire from './pages/retire.js';
+import share from './pages/share.js';
+
+es6Promise.polyfill();
+Raven.config(config.ravenPublicDSN).install();
+
+const routes = {
+  error,
+  experiment,
+  experiments,
+  home,
+  legacy,
+  onboarding,
+  retire,
+  share
+};
+
+const spl = window.location.pathname.split('/');
```

Comment:
We probably want to trim off & ignore trailing slashes here, too. (e.g. `/experiments` vs `/experiments/` and `/experiments/snooze-tabs` vs `/experiments/snooze-tabs/`